runtime/altair/Cargo.toml

```diff
@@ -157,6 +157,7 @@ std = [
   "serde/std",
   "log/std",
   # Substrate related
+  "substrate-wasm-builder",
```

Comment:
I think it should be `"substrate-wasm-builder/std"` to enable also std, if not, only the basic crate is added